src/booking.scss

```diff
@@ -1,299 +1,300 @@
+        padding: 5px;
+        border-radius: 5px;
+        border: solid lightgrey;
+        max-width: 200px;
     }
-  }
 
-  .second-step {
-    gap: 10px;
-  }
+    &__button {
+        background-color: #e3170a;
+        border: none;
+        border-radius: 4px;
+        color: #ffffff;
+        cursor: pointer;
+        font-size: 0.7rem;
+        font-weight: 700;
+        height: 66px;
+        width: 80%;
+        max-width: 308px;
+        float: right;
+        margin: 20px;
+        @media screen and (max-width: 500px) {
+            align-self: center;
+            margin-right: 0;
+            margin-left: 0;
+        }
+    }
 
-  .invisible {
-    display: none;
-  }
+    &__step-two {
+        position: absolute;
+        inset: 50%;
+        transform: translate(-50%, -50%);
+        background-color: white;
+        width: 90%;
+        height: 90%;
+        display: grid;
+        grid-template-rows: 1fr 1fr;
+        border: 2px solid gray;
+        overflow: auto;
+        @include breakpoint($xl) {
+            width: 1150px;
+        }
+    }
 
-  &__select {
-    width: 300px;
-  }
+    .second-step {
+        gap: 10px;
+    }
 
-  &__name-input {
-    width: 80%;
-    max-width: 300px;
-    height: 30px;
-    padding: 5px;
-    border-radius: 5px;
-    border: solid lightgrey;
-  }
-
-  &__e-mail-input {
-    width: 80%;
-    max-width: 300px;
-    height: 30px;
-    padding: 5px;
-    border-radius: 5px;
-    border: solid lightgrey;
-  }
-
-  &__select {
-    width: 200px;
-    height: 40px;
-    padding: 5px;
-    border-radius: 5px;
-    border: solid lightgrey;
-  }
-
-  &__submit-button {
-    background-color: #e3170a;
-    border: none;
-    border-radius: 4px;
-    color: #ffffff;
-    cursor: pointer;
-    font-size: 0.7rem;
-    font-weight: 700;
-    height: 66px;
-    width: 80%;
-    max-width: 308px;
-    float: right;
-    margin: 20px;
-    @media screen and (max-width: 500px) {
-      align-self: center;
-      margin-right: 0;
-      margin-left: 0;
+    .invisible {
+        display: none;
     }
-  }
 
-  &__step-three {
-    position: absolute;
-    inset: 50%;
-    transform: translate(-50%, -50%);
-    background-color: white;
-    width: 90%;
-    height: 90%;
-    border: 2px solid gray;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    @include breakpoint($xl) {
-      width: 1150px;
+    &__name-input {
+        width: 80%;
+        max-width: 300px;
+        height: 30px;
+        padding: 5px;
+        border-radius: 5px;
+        border: solid lightgrey;
     }
-  }
 
-  &__button-div {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-end;
-    justify-content: flex-end;
-    height: 1fr;
+    &__e-mail-input {
+        width: 80%;
+        max-width: 300px;
+        height: 30px;
+        padding: 5px;
+        border-radius: 5px;
+        border: solid lightgrey;
+    }
+
+    &__select {
```

Comment:
Missing width: 80%

Reply:
Good catch! Fixed that in the latest commit.